### ISSUE TITLE
Oxygen blocks are all fullwidth in Gutenberg Editor

### ISSUE THREAD

**Issue by yankiara** [soflyy/oxygen-bugs-and-features#743]
**Describe the bug**
When inserting an Oxygen block in Gutenberg Editor, block is full-width.
When inserting inside Gutenberg columns, it messes gutters up.
It's OK on frontend.

**Have you tried all the steps at https://oxygenbuilder.com/documentation/troubleshooting/troubleshooting-guide/?**
*Yes*

**Are you able to replicate the issue on a Sandbox install at https://oxygenbuilder.com/try?**
*yes: http://hungry-squirrel.w6.wpsandbox.pro*

**To Reproduce**
1. Add any Oxygen block to block library (for instance just text widget)
2. Edit Sample Page with Gutenberg
3. Insert the newly added block
4. Check block's width
5. Insert columns
6. Insert the block inside two adjacent columns
7. Check gutters

**Expected behavior**
Block should respect gutenberg page-width/margins/paddings when alone or inside a column.

**What actually happens**
When alone, the block is fullwidth, and when inside a colum, gutters are wrong, because of this CSS (added by [data-align="full"]):
```
.block-editor-block-list__layout .block-editor-block-list__block[data-align="full"] {
    margin-left: -45px;
    margin-right: -45px;
}
```

**Screenshots**
![fullwidth columns](https://user-images.githubusercontent.com/36680757/69497933-6948d500-0ee2-11ea-99f4-8d0e4b2fa15c.png)

**Desktop (please complete the following information):**
- OS: Windows
 - Browser: All
 - Oxygen Version 3.1

**Comment by VF404** [comment]
@yankiara 

could fix the 100% width problem to get the preview in "center view":
edit the lines of the oxygen-gutenberg.js (version 1.3) file (locate in Plugins => oxygen-gutenberg

line 50:
from: attributes['align'] = {type: 'string', default: 'full'};
to: attributes['align'] = {type: 'string', default: 'center'};

line 65:
from: if( 'full' != props.attributes.align ){ props.setAttributes({ align: 'full' }); }
to: if( 'full' != props.attributes.align ){ props.setAttributes({ align: 'center' }); }

BONUS: If you wanna register more width-options into oxygen gutenberg blocks (not tested yet the frontpage output):
line 58:
from:  align: ['full']
to: align: ['full', 'center', 'left', 'right']

**Comment by AnthonyBrowneCreative** [comment]
Yes, the same problem is in version 4.6.2. It would be great if the Oxygen Team could bake this fix into the source code so that we don't have to re-hack it after every update.
